src/routes/Overview.js

```diff
@@ -64,31 +61,16 @@ const Overview = ({ loadApis, services, history, selectRow, onError }) => {
                                                 />
                                             </LevelItem>
                                             <LevelItem>
-                                                <Dropdown
-                                                    dropdownItems={ [
-                                                        <DropdownItem
-                                                            key="download"
-                                                            component="button"
-                                                            onClick={ () => {
-                                                                multiDownload(services.selectedRows, onError);
-                                                            } }
-                                                        >
-                                                            Download selected
-                                                        </DropdownItem>
-                                                    ] }
-                                                    isOpen={ isOpen }
-                                                    onSelect={ () => onOpenToggle(!isOpen) }
-                                                    toggle={
-                                                        <KebabToggle
-                                                            onToggle={ (isOpen) => onOpenToggle(isOpen) }
-                                                            isDisabled={ !services.selectedRows ||
-                                                                Object.values(services.selectedRows || {})
-                                                                .map(({ isSelected }) => isSelected)
-                                                                .filter(Boolean).length === 0 }
-                                                        />
-                                                    }
-                                                    isPlain
-                                                />
+                                                <Button
+                                                    isDisabled={ !services.selectedRows ||
+                                                        Object.values(services.selectedRows || {})
+                                                        .map(({ isSelected }) => isSelected)
+                                                        .filter(Boolean).length === 0 }
+                                                    onClick={ () => multiDownload(services.selectedRows, onError) }
+                                                    style= { { marginLeft: '1em' } }
```

Comment:
Inline styles are not really good, it will be better to add `className="ins-c-api__download-selected"` here and in `src/App.scss` add
```SCSS
.ins-c-api__download-selected {
  margin-left: var(--pf-global--spacer--md);
}
```

Reply:
Got it! Thanks a lot! Was skeptical about the inline styling as well